app/uk/gov/hmrc/apigatekeeperorganisationfrontend/views/UpdateSubmissionPage.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.apigatekeeperorganisationfrontend.controllers.UpdateSubmissionController.{UpdateSubmissionViewModel, UpdateSubmissionForm}
+@import uk.gov.hmrc.govukfrontend.views.html.components.{BackLink, GovukTextarea, Text, Textarea}
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.apigatekeeperorganisationfrontend.controllers.routes
+
+@import java.time.ZoneOffset
+@import java.time.format.DateTimeFormatter
+
+@this(layout: Layout, govukTextarea: GovukTextarea, govukButton: GovukButton)
+
+@(model: UpdateSubmissionViewModel, form: Form[UpdateSubmissionForm])(implicit request: RequestHeader, messages: Messages)
+
+@layout(
+  pageTitle = Some("HMRC API Gatekeeper - Add a comment"),
+  backLink = Some(BackLink.mimicsBrowserBackButtonViaJavaScript)
+) {
+
+    <span class="govuk-caption-l">@{messages("updatesubmission.subtitle")}</span>
+    <h1 class="govuk-heading-l">@{messages("updatesubmission.title", model.organisationName.toString())}</h1>
+
+    @helper.form(action = routes.UpdateSubmissionController.action(model.submissionId, model.instanceIndex)) {
+      @helper.CSRF.formField
+
+        @govukTextarea(Textarea(
```

Comment:
maybe worth using to show the max 500 https://design-system.service.gov.uk/components/character-count/ ?

Reply:
Good shout - done!